src/PKSim.Infrastructure/Services/ImportObservedDataTask.cs

```diff
@@ -306,13 +306,13 @@ public IEnumerable<string> PredefinedValuesFor(string name)
          return Enumerable.Empty<string>();
       }
 
-      public IReadOnlyList<string> DefaultMetaDataCategories => CoreConstants.ObservedData.DefaultProperties;
+      public IReadOnlyList<string> DefaultMetaDataCategories { get; } = CoreConstants.ObservedData.DefaultProperties;
 
-      public IReadOnlyList<string> ReadOnlyMetaDataCategories => new List<string> { };
+      public IReadOnlyList<string> ReadOnlyMetaDataCategories { get; } = new List<string> { };
+    
+      public bool MolWeightAlwaysEditable { get; } = false;
```

Comment:
@georgeDaskalakis I changed this stuff here. Also when you always return a constant, use {get;} instead 0f => false. A bit faster...